packages/eslint-plugin-bigtest/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "eslint-plugin-bigtest",
```

Comment:
We should stick with the `@bigtest/eslint-plugin` convention

Reply:
renamed